moshi/src/main/java/com/squareup/moshi/LinkedHashTreeMap.java

```diff
@@ -44,18 +44,18 @@ public int compare(Comparable a, Comparable b) {
     }
   };
 
-  Comparator<? super K> comparator;
-  Node<K, V>[] table;
-  final Node<K, V> header;
+  private Comparator<? super K> comparator;
+  private Node<K, V>[] table;
+  private final Node<K, V> header;
   int size = 0;
-  int modCount = 0;
-  int threshold;
+  private int modCount = 0;
```

Comment:
`table` and `modCount` are accessed by the inner class `LinkedTreeMapIterator`, so synthetic accessor methods would be created for them if set to private.
Also, I think this class is copied from Gson, so it's probably best not to modify it here. (Just imo, I don't know.)